packages/manager/modules/exchange/src/account/add/account-add.controller.js

```diff
@@ -185,7 +185,7 @@ export default class ExchangeAccountAddController {
       lastName: this.newAccount.lastName,
       displayName: this.newAccount.displayName,
       initials: this.newAccount.initials,
-      description: this.newAccount.description,
+      description: `"${this.newAccount.description}"`,
```

Comment:
`${this.newAccount.description}` returns new string. Why do you want to add ""?

Reply:
Because there is an option which allow to generate a CSV file.
And for decription we can have those type of separtors (`, ; :`) in description.
Those chars had to be ignored to avoid generating CSV file.